src/Compiler/Utilities/illib.fs

```diff
@@ -137,7 +137,7 @@ module internal PervasiveAutoOpens =
     type Async with
 
         static member RunImmediate(computation: Async<'T>, ?cancellationToken) =
-            let cancellationToken = defaultArg cancellationToken Async.DefaultCancellationToken
+            let cancellationToken = defaultArg cancellationToken CancellationToken.None
```

Comment:
What's the reason for this change? It might be a good idea in general, just asking how did you stumble on it.

Reply:
This is out of desperation. I'm seeing OperationCancelledException in various test cases. We do a lot of RunSychronously / RunImmediate in the tests but also in FCS. I have no idea how to debug this, because it's very random, but my intuition is that Async.CancelDefaultToken() called in some test cases kills the asyncs in the same process? I have no idea if this makes sense, so this one is just experimental.